XPlat/DropboxApiV2/component/License.md

```diff
@@ -0,0 +1,18 @@
+**Xamarin is not responsible for, nor does it grant any licenses to, third-party packages. Some packages may require or install dependencies which are governed by additional licenses.**
```

Comment:
Move this to the root.  XPlat/DropboxApiV2 and fix the component yaml to reference the new location.

Reply:
Moved the file and updated the yaml references